pkg/command/plugin_bundle.go

```diff
@@ -35,6 +37,7 @@ func newDownloadBundlePluginCmd() *cobra.Command { // nolint:dupl
 	f := downloadBundleCmd.Flags()
 	f.StringVarP(&dpbo.pluginDiscoveryOCIImage, "image", "", "", "URI of the plugin discovery image providing the plugins")
```

Comment:
Should this have a default value to point to default plugin discovery? cc @marckhouzam @vuil

Reply:
Good idea.  I think it should.  Since pretty much every download-bundle will be from the central repo, might as well make it the default. 👍